backend/data/src/main/kotlin/io/tolgee/service/label/LabelService.kt

```diff
@@ -0,0 +1,61 @@
+package io.tolgee.service.label
+
+import io.tolgee.dtos.request.label.CreateLabelDto
+import io.tolgee.model.Project
+import io.tolgee.model.translation.Label
+import io.tolgee.repository.LabelRepository
+import jakarta.persistence.EntityManager
+import jakarta.transaction.Transactional
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+import org.springframework.stereotype.Service
+import java.util.Optional
+
+@Service
+class LabelService(
+  private val labelRepository: LabelRepository,
+  private val entityManager: EntityManager,
+  ) {
+  fun getProjectLabels(projectId: Long, pageable: Pageable): Page<Label> {
+    return labelRepository.findByProjectId(projectId, pageable)
+  }
+
+  fun find(labelId: Long): Optional<Label> {
+    return labelRepository.findById(labelId)
+  }
+
+  @Transactional
+  fun createLabel(
+    projectId: Long,
+    dto: CreateLabelDto,
+    ): Label {
+    val label = Label()
+
+    label.name = dto.name
```

Comment:
don't duplicate the assigning, create separate method for it and use it in both create and update